src/style.css

```
@@ -0,0 +1,157 @@
+/* #region card */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+
+  background: var(--white);
+  border: 1px solid var(--grey);
+  border-radius: 5px;
+
+  --white: #fff;
+  --grey: #f3f3f3;
+  --dark-grey: #616070;
+  --light-blue: #00acdc;
+  --dark-blue: #060b35;
```

```suggestion
  --white: #fff;
  --grey: #f3f3f3;
  --dark-grey: #616070;
  --light-blue: #00acdc;
  --dark-blue: #060b35;
```
Move it under the `:root` selector